bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/monoHiggs/2HDM+a/sinp_0p35_tanb_1_mXd_10/bbProduction/bbProduction_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model Pseudoscalar_2HDM
+define p = p b b~
```

Comment:
one last thing, we recommend to remove the proton and jet definition,
as this is taken care automatically depending on the input model

Reply:
@perrozzi 
  My understanding is that the flavor scheme is automatically taken care of only in the gridpack_generation.sh in the master branch.

While in my case, I am generating samples for EXO-16-050, a 2016 analysis and Moriond 2017 MC samples. To be consistent with other mono-Higgs samples, I am using Madgraph 2.4.2 and the gridpack_generation.sh in mg240 branch.

Note, I did ask in the past if I should commit this kind of MC cards in mg240 branch or master branch in the pre-2017 folder. I was told to commit in the pre-2017 folder.

Reply:
@perrozi,

 Hello, are the cards OK? Can they be merged?

Reply:
Hi Eiko, I think is not a good idea to use the mg240 branch but the master one instead, even if the samples are meant to be used for 2016 MC (in any case you would get MG5_aMC v2.4.2 but in the mg240 branch you would not get all the updates and fixes to the production scripts etc).